packages/components/src/sandbox/index.native.js

```diff
@@ -117,7 +157,6 @@ function Sandbox( {
 	url,
 } ) {
 	const ref = useRef();
-	const [ width, setWidth ] = useState( 0 );
```

Comment:
We really only need to calculate the height as the width should fit the block's size, so I removed the width calculation.